mne/beamformer/_lcmv.py

```diff
@@ -237,6 +255,23 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
 
     del Cm
 
+    # leadfield rank and optional rank reduction
+    if reduce_rank is not False:
+        if not pick_ori == 'max-power':
+            raise NotImplementedError('The computation of a rank reduced '
+                                      'leadfield with reduce_rank is not yet '
+                                      'implemented with free or fixed '
+                                      'orientation of the forward solution.')
+        if isinstance(reduce_rank, bool):
+            rank_G = estimate_rank(G[:, 0:3], tol='auto') - 1
```

Comment:
arfff this is an auto here. I suggest to remove it.

btw I taking over the PR so this can be merged quickly.